web/frontend/src/components/slam-2d-render.vue

```diff
@@ -39,9 +63,14 @@ const props = defineProps<{
   resources: commonApi.ResourceName.AsObject[]
   pointcloud?: Uint8Array
   pose?: commonApi.Pose
+  destExists?: boolean
```

Comment:
Why are these nullable? Could we make them non nullable?

Reply:
you're right. there is no reason for them to nullable since they are explicitly defined in slam.vue

will make them non nullable.